src/client/Client.cc

```diff
@@ -4250,7 +4250,7 @@ void Client::trim_caps(MetaSession *s, uint64_t max)
       // disposable non-auth cap
       if (!(get_caps_used(in.get()) & ~oissued & mine)) {
 	ldout(cct, 20) << " removing unused, unneeded non-auth cap on " << *in << dendl;
-	cap = (remove_cap(cap, true), nullptr);
+	remove_cap(cap, true);
```

Comment:
This is just defensive code to explicit note that cap is no longer valid. Let's keep it please.

Reply:
ok, I close it